### PR TITLE
Release

### PR DESCRIPTION
**Pull request by lbetz**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     },
     {
       "name": "icinga/icinga",
-      "version_requirement": ">= 2.9.0 < 6.0.0"
+      "version_requirement": ">= 2.9.0 < 7.0.0"
```
Comment (bastelfreak) [inline metadata.json:21]: I think we should do an individual PR for this, so it appears in the changelog and people recognize this was changed

File: metadata.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-icinga2",
-  "version": "5.0.2-rc0",
+  "version": "6.0.1-rc0",
```
Comment (bastelfreak) [inline metadata.json:3]: If you want to do a 6.0.0 release, please update this to 6.0.0
```suggestion
  "version": "6.0.0",
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,29 @@ All notable changes to this project will be documented in this file.
...
+**Breaking changes:**
+
+- Remove EL7 support [\#774](https://github.com/voxpupuli/puppet-icinga2/pull/774) ([lbetz](https://github.com/lbetz))
+- Remove Debian Buster support [\#773](https://github.com/voxpupuli/puppet-icinga2/pull/773) ([lbetz](https://github.com/lbetz))
+
+**Implemented enhancements:**
+
+- Add Ubuntu noble \(24.04\) support [\#781](https://github.com/voxpupuli/puppet-icinga2/pull/781) ([lbetz](https://github.com/lbetz))
+- Add support for Fedora 40 [\#775](https://github.com/voxpupuli/puppet-icinga2/pull/775) ([lbetz](https://github.com/lbetz))
+- Fix SELinux and add feature::api::bind\_port support [\#772](https://github.com/voxpupuli/puppet-icinga2/pull/772) ([lbetz](https://github.com/lbetz))
```
Comment (bastelfreak) [inline CHANGELOG.md:20]: is that an enhancement or a bugfix?

Comment (lbetz) [reply CHANGELOG.md:20]: Both. Fixes missing sel_type of file resources. To change the listening port, it's a feature.

**Pull request closed without merging**